src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -27,12 +28,22 @@ const TopicDomainHeader = (props) => {
       <Pressable onPress={handlePress} style={{backgroundColor: 'transparent'}}>
         <View style={{flexDirection: 'row', alignItems: 'center'}}>
           <FastImage testID="imageTopicMember" source={TopicMemberIcon} style={styles.member} />
-          <Text style={styles.domainMember}>{memberCount} Members</Text>
+          {props.isLoading ? (
+            <Shimmer height={10} width={normalize(25)} />
+          ) : (
+            <Text style={styles.domainMember}>{memberCount}</Text>
+          )}
+          <Text style={styles.domainMember}> Members</Text>
         </View>
-        {isFollow && !hideSeeMember && (
-          <Text style={styles.seeMemberText} numberOfLines={1} ellipsizeMode="tail">
-            See community member
-          </Text>
+        {props.isLoading ? (
+          <Shimmer height={10} width={normalize(60)} />
+        ) : (
+          isFollow &&
+          !hideSeeMember && (
+            <Text style={styles.seeMemberText} numberOfLines={1} ellipsizeMode="tail">
+              See community member
+            </Text>
+          )
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The conditional rendering based on `props.isLoading` is a good approach to handle loading states. However, it's important to ensure that the `isLoading` prop is always passed to this component to avoid unexpected behavior. If not provided, JavaScript will treat it as `undefined`, which is a falsy value and might lead to incorrect rendering.



<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
it's always provided